music_rnn/src/model.ts

```diff
@@ -27,10 +27,11 @@ const CELL_FORMAT = 'multi_rnn_cell/cell_%d/basic_lstm_cell/';
  *
  * A MusicRNN is an LSTM-based language model for musical notes.
  */
-export class MusicRNN {
+export class MusicRNN<T extends magenta.controls.ControlSignalUserArgs> {
```

Comment:
Maybe we should use a different name than "T" to make it more clear that this relates to the control sequence type.

Reply:
I couldn't find anything about this in a TypeScript style guide; in most languages T is pretty idiomatic here (or sometimes K for keys, V for values, etc.).  Maybe I'll use A for args.